lib-py/casmutils/xtal/coordinate.py

```diff
@@ -79,12 +140,42 @@ def __ne__(self, other):
         """
         return not self==other
 
+    def __add__(self, other):
+        """Adds the "other" value to the Coordinate instance
+
+        Parameters
+        ----------
+        other : Coordinate
+
+        Returns
+        -------
+        Coordinate
+
+        """
+        py_binded = self._pybind_value + other._pybind_value
+        return self._from_pybind(py_binded)
```

Comment:
Documentation says this will return `_Coordinate`, which we don't want. Either make sure the behavior is correct, or update the documentation.